A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/Clock/A320_Neo_Clock.js

```diff
@@ -57,10 +60,25 @@ class A320_Neo_Clock extends BaseAirliners {
             }
 
             if (this.middleSelectorElem) {
-                const currentDisplayTime = SimVar.GetSimVarValue("L:PUSH_CHRONO_SET", "Bool") ? this.getUTCDate() : this.getUTCTime()
+                let currentDisplayTime = "";
+                let currentDisplayTime2 = "";
+                if (SimVar.GetSimVarValue("L:PUSH_CHRONO_SET", "Bool") == true) {
+
```

Comment:
```suggestion
```

Reply:
This was from a previous PR, not my part of code, but OK